### PR TITLE
Add Fineco

### LINKED ISSUE #8942: Missing fineco icon
### Brand Name

Fineco

### Website

Www.fineco.it

### Popularity Metric

https://www.similarweb.com/website/finecobank.com/#overview 

### Official Resources for Icon and Color

None

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by nicolgit**
**Issue:** closes #8942 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
nothing to say :-)

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the contribution @nicolgit - though I'm spotting a few [differences](https://github.com/simple-icons/simple-icons/assets/1830867/055b2275-91a6-4a10-a8fa-f0ee67193a8b), with your PR in black on top of the header SVG from the website.
Would you mind taking another look? 🙏

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4456,6 +4456,11 @@
             "hex": "BF0000",
             "source": "https://commons.wikimedia.org/wiki/File:FileZilla_logo.svg"
         },
+        {
+            "title": "Fineco",
+            "hex": "155BA0",
+            "source": "https://commons.wikimedia.org/wiki/File:FinecoBank_logo.svg"
```
[the remark this answers is not in the document]
Comment (nicolgit) [reply _data/simple-icons.json]: ok!

File: _data/simple-icons.json [hunk 2]
```diff
@@ -4456,6 +4456,11 @@
             "hex": "BF0000",
             "source": "https://commons.wikimedia.org/wiki/File:FileZilla_logo.svg"
         },
+        {
+            "title": "Fineco",
+            "hex": "155BA0",
```
[the remark this answers is not in the document]
Comment (nicolgit) [reply _data/simple-icons.json]: ok!

**Comment by adamrusted** [comment]
There were the differences I'd identified in the linked screenshot - so might be worth regenerating from the new source now you've updated it. Worst case I can lend a hand with the svg path but we'd need to get another maintainer to review.

**Comment by nicolgit** [comment]
rebuilt the svg starting from https://[images.finecobank.com/common-global/img/logo/logo-fineco.svg](https://images.finecobank.com/common-global/img/logo/logo-fineco.svg)

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**